### PR TITLE
fixes for using alternate PXR_NS

### PR DESCRIPTION
**Pull request by elrond79**

### CODE REVIEW

**Comment by kxl-adsk** [comment]
I don't see anything wrong in this change and we discussed it internally as well. Since it impacts AL and Pixar plugins, I would like to see thumbs up from @mattyjams and @murphyeoin before merging.

File: plugin/al/usdtransaction/AL/usd/transaction/Transaction.cpp [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
 #include <iostream>
-using namespace pxr;
+PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace AL {
 namespace usd {
 namespace transaction {
 
-Transaction::Transaction(const pxr::UsdStageWeakPtr& stage, const pxr::SdfLayerHandle& layer)
+Transaction::Transaction(const PXR_NS::UsdStageWeakPtr& stage, const PXR_NS::SdfLayerHandle& layer)
```
Comment (mattyjams) [inline plugin/al/usdtransaction/AL/usd/transaction/Transaction.cpp]: With the PXR_NAMESPACE_USING_DIRECTIVE added above, we should be able to just drop the "PXR_NS::" off of these.

File: plugin/pxr/pxr/pxr.h.in [hunk 2]
```diff
@@ -1,60 +0,0 @@
-//
```
Comment (mattyjams) [inline plugin/pxr/pxr/pxr.h.in]: Thanks for removing this! I wasn't sure why it was still here either.

File: plugin/al/usdtransaction/AL/usd/transaction/Transaction.cpp (hunk 1 above)
Comment (elrond79) [reply plugin/al/usdtransaction/AL/usd/transaction/Transaction.cpp]: Good catch - was just doing find/replace, should have been checking if they were needed in the first place!  Fixed now.

File: plugin/pxr/pxr/pxr.h.in (hunk 2 above)
Comment (elrond79) [reply plugin/pxr/pxr/pxr.h.in]: I went through and added pxr.h everywhere the namespace stuff was in use.  And yeah, I think the pxr.h.in in THIS repo was probably just leftover cruft...

**Comment by elrond79** [comment]
Updated to address Matt's notes.. thanks!

**Comment by elrond79** [comment]
Are we waiting for AL to ok this?

**Comment by elrond79** [comment]
Awesome, thanks!

**Pull request merged**